dev/conductor/core/lib/src/proto/compile_proto.sh

```diff
@@ -7,7 +7,7 @@
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 # Ensure dart-sdk is cached
-"$DIR/../../../../bin/dart" --version
+# "$DIR/../../../../bin/dart" --version
```

Comment:
why did you comment this out?

Reply:
Was supposed to revert this, thanks for catching this!  

I was having issues getting the protos to compile, I was commenting lines out to get a good error readout.  This should have gone back to its original state after I solved the problem.

Reply:
It's coming back to me, I would have liked to mute this output, however `dart --version` prints the version to STDERR, and I didn't want to mute STDERR in case there was a compilation error or something.